specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/1.2-preview/iotcentral.json

```diff
@@ -0,0 +1,5641 @@
+            "$ref": "./examples/dataexports_destinations_get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "$ref": "#/parameters/DestinationId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Destination"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "Destinations_Create",
+        "summary": "Create or update a destination",
+        "description": "Create or update a definition for where to send data.",
+        "x-ms-examples": {
+          "Create a webhook destination": {
+            "$ref": "./examples/dataexports_destinations_create_webhook.json"
+          },
+          "Create an azure blob storage destination": {
+            "$ref": "./examples/dataexports_destinations_create_blob.json"
+          },
+          "Create an azure data explorer destination": {
+            "$ref": "./examples/dataexports_destinations_create_adx.json"
+          },
+          "Create an azure event hub destination": {
+            "$ref": "./examples/dataexports_destinations_create_eventhub.json"
+          },
+          "Create an azure service bus queue destination": {
+            "$ref": "./examples/dataexports_destinations_create_queue.json"
+          },
+          "Create an azure service bus topic destination": {
+            "$ref": "./examples/dataexports_destinations_create_topic.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "$ref": "#/parameters/DestinationId"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Destination body.",
+            "schema": {
+              "$ref": "#/definitions/Destination"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Destination"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "patch": {
+        "operationId": "Destinations_Update",
+        "summary": "Patch a destination.",
+        "description": "Perform an incremental update to a destination.",
+        "x-ms-examples": {
+          "Update an webhook destination": {
+            "$ref": "./examples/dataexports_destinations_update_webhook.json"
+          }
+        },
+        "consumes": [
+          "application/json",
```

Comment:
What does it mean to accept "application/json" to this patch operation?  Does it act like a JSON merge patch in this case?  Or something else?

Reply:
We kept the old one previously to avoid breaking customer who still use the application/json, backend code handling the payload in json patch way.

Reply:
But this is a new endpoint.

If the body must be a json patch payload, regardless of content type, then you should document that clearly in the description of the body parameter.

This isn't necessary for this preview but please consider fixing this for your GA.

Reply:
Seems it's wrong on our side by using the json-patch+json, the backend actually throws 400 when passing json-patch format, we are currently using the json-merge-patch+json.